native-schema-registry/python/PyGsrSerDe/test_AvroSerDe.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from AvroSerDe import *
+
+
+class TestExample(unittest.TestCase):
+    def test_example(self):
+        # sample test method to check if unittests are working
+        self.assertEqual('a', 'a')
+
+
+class TestAvroSerDe(unittest.TestCase):
+
+    def test_serde(self):
```

Comment:
Can we add more comprehensive test cases with additional test coverage. e.g. Serialize/de-serialize when record/schema is null or the table is empty. Serialize/de-serialize for schema contains enum/array/object/union/union with null/fixed array/map/all types. Serialize with schema that contains unknown type and/or invalid schema.

for reference: https://github.com/blacktooth/aws-glue-schema-registry/blob/native-schema-registry/native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe.Tests/deserializer/avro/AvroDeserializerTest.cs

Reply:
Updated with test cases for many of the types listed.